rest_framework_gis/serializers.py

```diff
@@ -118,7 +123,7 @@ def to_representation(self, instance):
         processed_fields = set()
 
         # optional id attribute
-        if self.Meta.id_field:
+        if self.Meta.id_field is not None:
```

Comment:
```suggestion
        if self.Meta.id_field:
```
This somehow slipped in the force-pushed commit. In the PR I quoted earlier https://github.com/3nids/django-rest-framework-gis/pull/2, this line is not modified and is as in the suggestion.

Note this is required by the exact test that is failing, namely `test_geojson_false_id_attribute_slug` which tests when the `id_field = False`. In that case having a strict check like this `self.Meta.id_field is not None` will fail and cause the serializer to search for field named `False`.